media/js/firefox/os/desktop.js

```diff
@@ -233,7 +233,11 @@
     });
 
     //track GA event for newsletter CTA
-    trackGAEvent(['_trackEvent', 'FxOs Consumer Page', 'click', cta]);
+    window.dataLayer.push({
```

Comment:
This file is not being used in production and will be removed in #2995, is it worth even changing here?